v2/api_error_test.go

```diff
@@ -151,6 +151,49 @@ func TestGRPCStatus(t *testing.T) {
 	}
 }
 
+func TestReason(t *testing.T) {
+	tests := []struct {
+		ei *errdetails.ErrorInfo
+	}{
+		{&errdetails.ErrorInfo{Reason: "Foo", Domain: "Bar", Metadata: map[string]string{"type": "test"}}},
+		{&errdetails.ErrorInfo{}},
+	}
+	for _, tc := range tests {
+		apierr := make(tc.ei)
+		if diff := cmp.Diff(apierr.Reason(), tc.ei.GetReason()); diff != "" {
+			t.Errorf("got(+), want(-),: \n%s", diff)
```

Comment:
I believe `got` is `-` and `want` is `+`?

Reply:
I followed this link Cody sent and it has it the opposite way: https://pkg.go.dev/github.com/google/go-cmp/cmp#example-Diff-Testing